lib/gracenote/request/search.rb

```diff
@@ -0,0 +1,29 @@
+require "gracenote/request/option"
+
+module Gracenote
+  module Request
+    module Search
+      # @params mode [String] 'SINGLE_BEST' or 'SINGLE_BEST_COVER'
+      # @params range [Hash] { start: 1, end: 10 }
+      #
+      def search(artist: nil, album: nil, track: nil, mode: nil, range: nil)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for search is too high. [22.23/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Cyclomatic complexity for search is too high. [8/6]
* Method has too many lines. [15/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)
* Perceived complexity for search is too high. [8/7]